docs/help/_posts/2010-01-02-creating-a-substitute.markdown

```diff
@@ -15,7 +15,7 @@ This is how you'll normally create substitutes for types. Generally this type wi
 
 ⚠️ **Warning:** Substituting for classes can have some nasty side-effects!
 
-For starters, **NSubstitute can only work with *virtual* members of the class** that are overridable in the test assembly, so any non-virtual code in the class will actually execute! If you try to substitute for a class that formats your hard drive in the constructor or in a non-virtual property setter then you're asking for trouble. (By overridable we mean `public virtual`, `protected virtual`, `protected internal virtual`, or `internal virtual` with `InternalsVisibleTo` attribute applied. See [How NSubstitute works](/help/how-nsub-works) for more information.)
```

Comment:
This still has `protected virtual` in the doc. I thought we wanted to remove it. Am I missing something?

Reply:
I admit this is confusing. These docs are referring to real code executing for non-overridable members, which includes protected virtual code, even if these calls are not (easily) configurable via the NSubstitute API. It's not about what methods we can configure with NSub -- it's about what real code can execute.

There is probably a better way to communicate this. 🤔